### PR TITLE
Update contributing guide

### PR DESCRIPTION
**Pull request by jmcook1186**
### Types of changes

- [ ] Enhancement (project structure, spelling, grammar, formatting)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### A description of the changes proposed in the Pull Request
Updates contribution guide for latest IF

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,145 +8,128 @@ The following document is a rule set of guidelines for contributing.
...
-      
-      Examples:
-      
-      ```    
-        - remove deprecated logger
-        - refactor some method
-        - add JSDoc on existing function
-      ```
+Pull requests will not be reviewed unless they pass all CI. This includes a lint check and running our unit tests.
+
```
Comment (narekhovhannisyan) [inline CONTRIBUTING.md:100]: but we still need commit message guidelines

Comment (narekhovhannisyan) [reply CONTRIBUTING.md:100]: it's done above no worries

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -8,145 +8,128 @@ The following document is a rule set of guidelines for contributing.
...
-   or your code is unlikely be merged into the main project.
+Run `npm run fix` before commiting. If your commit message does not conform to the conventional commit specification or if you have not run `npm run fix` your commit will not satisfy the commitlint check.
 
-#### Step 4: Sync
+Add and commit with your commit message:
 
-Use git pull/merge to synchronize your work with the IF repository.
+```bash
+$ git add my/changed/files
+$ git commit m "<my-commit-message>"
```
Comment (narekhovhannisyan) [inline CONTRIBUTING.md:84]: git commit m "<type-of-commit>(<my-optional-scope>): <my-commit-message>"

**Review by narekhovhannisyan — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (jmcook1186) [reply CONTRIBUTING.md:84]:
```suggestion
$ git commit -m "<type-of-commit>(<my-optional-scope>): <my-commit-message>"
```

**Pull request merged**